File: official/docs/csharp/batches/AddShipmentsToBatch.cs
using EasyPost;

EasyPost.ClientManager.SetCurrent("EASYPOST_API_KEY");

Batch batch = await Batch.Retrieve("batch_...");

batch.AddShipments("shipments", new List<Dictionary<string, object>>(){
    new Dictionary<string, object>(){
        { "id", "shp_..."},
        { "id", "shp_..."}
     }
    }
 );

Console.WriteLine(JsonConvert.SerializeObject(batch, Formatting.Indented));

File: official/docs/node/batches/add_shipments_to_batch.js
const Easypost = require('@easypost/api');

const api = new Easypost(process.env.EASYPOST_API_KEY);

const batch = api.Batch.retrieve('batch_...');

batch.addShipments([{ id: 'shp_...' }, { id: 'shp_...' }]).then(console.log);
